src/pages/admin/AdminUsersList.jsx

```
@@ -5,30 +5,173 @@ import styles from './AdminUsersList.module.scss';
+        // 3. res 자체가 배열인 경우
+        else if (Array.isArray(res)) {
+          userData = res;
+          pageInfo.totalElements = userData.length;
+          console.log('✅ res 배열 구조로 처리:', userData.length, '개');
+        }
+        // 4. res.data.content 구조인 경우
+        else if (res.data && res.data.content && Array.isArray(res.data.content)) {
+          userData = res.data.content;
+          pageInfo = {
+            number: res.data.number || 0,
+            totalPages: res.data.totalPages || 1,
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)